livestream-compose-sample/src/main/java/io/getstream/livestream/compose/streams/CameraLiveStream.kt

```diff
@@ -46,49 +54,81 @@ fun CameraLiveStream(
     val context = LocalContext.current
     val cameraProviderFuture = remember { ProcessCameraProvider.getInstance(context) }
 
-    LaunchedEffect(Unit) {
-        listViewModel.start()
-    }
-
-    ChatTheme {
-        Box(modifier = Modifier.fillMaxSize()) {
-            AndroidView(
-                factory = { ctx ->
-                    val preview = PreviewView(ctx)
-                    val executor = ContextCompat.getMainExecutor(ctx)
-                    cameraProviderFuture.addListener({
+    Box(modifier = Modifier.fillMaxSize()) {
+        AndroidView(
+            factory = { ctx ->
+                val preview = PreviewView(ctx)
+                val executor = ContextCompat.getMainExecutor(ctx)
+                cameraProviderFuture.addListener(
+                    {
                         val cameraProvider = cameraProviderFuture.get()
                         bindPreview(
                             lifecycleOwner,
                             preview,
                             cameraProvider
                         )
-                    }, executor)
-                    preview
-                },
-                modifier = Modifier.fillMaxSize(),
-            )
-            CommentsBox(
-                modifier = Modifier
-                    .background(
-                        brush = Brush.verticalGradient(
-                            listOf(Color.Transparent, Color.Black),
-                            0f,
-                            1050f,
+                    },
+                    executor
+                )
+                preview
+            },
+            modifier = Modifier.fillMaxSize()
+        )
+        //Gradient overlay
+        Spacer(
```

Comment:
A spacer is used for margins, not overlays. We should use something like a Box, Surface or just in general add modifiers to root components.

Reply:
Please help me break this down , in my testing i could not get same effect with spacer , i did borrow this thought process from a SO linked article

Reply:
Updated this with a `box`